modules/configuring-private-storage-endpoint-azure-vnet-subnet-iro-discovery.adoc

```diff
@@ -0,0 +1,141 @@
+[id="configuring-private-storage-endpoint-azure-vnet-subnet-iro-discovery"]
+= Setting up a private storage endpoint on Microsoft Azure by configuring the Image Registry Operator to discover VNet and subnet names
+
+The following procedure shows you how to set up a private storage endpoint on Microsoft Azure by configuring the Image Registry Operator to discover vNet and subnet names.
+
+.Prerequisites 
+
+* You have configured an image registry to run on Azure
+* You have configured a route
+
+.Procedure 
+
+. Edit the Image Registry Operator `config/instance` object and omit your VNet and subnet names:
++
+[source,terminal]
+----
+$ oc edit configs.imageregistry/cluster
+----
++
+[source,terminal]
+----
+# ...
+spec:
+  [...]
+   storage:
+      azure:
+        [...]
+        networkAccess:
+          type: Internal
+# ...
+----
+
+. Optional. Enter the following command to confirm that the Operator has completed provisioning. This might take a few minutes. 
++
+[source,terminal]
+----
+$ oc get configs.imageregistry/cluster -o=jsonpath="{.spec.storage.azure.privateEndpointName}" -w
+----
+
+////
+. Optional. Enter the following command to enable redirect, which redirects image pulls to the storage account. Note that with redirect enabled, pulling images is not expected to work.
++
+[source,terminal]
+----
+$ oc patch configs.imageregistry cluster --type=merge -p '{"spec":{"disableRedirect": false}}'
+----
+////
+
+.Verification
+
+Use the following verification steps to ensure that users within the same cluster can pull images from the registry.
+
+. Fetch the registry endpoint by running the following command:
++
+[source,terminal]
+----
+$ REGISTRY=$(oc get route default-route -n openshift-image-registry --template='{{ .spec.host }}')
+----
+
+. Enter the following command to log in to your container registry:
++
+[source,terminal]
+----
+$ podman login --tls-verify=false -u unused -p $(oc whoami -t) ${REGISTRY}
+----
++
+.Example output
++
+[source,terminal]
+----
+Login Succeeded!
+----
+
+. Enter the following command to verify that you can pull an image from the registry:
++
+[source,terminal]
+----
+$ podman pull --tls-verify=false ${REGISTRY}/openshift/tools
+----
++
+.Example output
++
+[source,terminal]
+----
+Trying to pull ${REGISTRY}/openshift/tools...
+Getting image source signatures
+Copying blob 6b245f040973 done
+Copying config 22667f5368 done
+Writing manifest to image destination
+Storing signatures
+22667f53682a2920948d19c7133ab1c9c3f745805c14125859d20cede07f11f9
+----
+
+
+. Optional. If the registry is exposed by a route, whether the `defaultRoute` or otherwise, disable `redirect` on the Image Operator configuration by entering the following command:
++
+[source,terminal]
+----
+$ oc patch configs.imageregistry cluster --type=merge -p '{"spec":{"disableRedirect": true}}'
+----
+
+.Verification with redirect enabled
+
+[NOTE]
+====
+With redirect enabled, pulling images is not expected to work.
+====
```

Comment:
It's still showing up under verification for me: https://github.com/openshift/openshift-docs/pull/69024/files#diff-0f818e6fea9e26ad4b280500ab5cc11f0ecd726705362ea4538f25d082c1e468R102
Am I missing something? maybe you forgot to push?

Reply:
You mean under procedure (instead of verification)? I think it's important to let customers know that if they expose a route to the registry and are configuring their storage account to be private that they will need to disable redirect if they want pulls external to the cluster to continue to work.

Reply:
looks good!